Dockerfile

```
@@ -23,8 +23,8 @@ RUN go mod download
 ADD . .
 RUN make
 
-FROM amazonlinux:2
-RUN yum install ca-certificates e2fsprogs xfsprogs util-linux -y
+FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
```

is there a multi arch image for this? I would prefer stick with multi-arch image to make multi arch image build easier.

---

`k8s.gcr.io/build-image/debian-base` (the same image but using the vanity domain) is multi arch.

```suggestion
FROM k8s.gcr.io/build-image/debian-base:v2.1.3
```

---

Thanks. I have done it.